cdc/processor/sinkmanager/table_sink_worker.go

```diff
@@ -142,31 +142,33 @@ func (w *sinkWorker) handleTask(ctx context.Context, task *sinkTask) (finalErr e
 			zap.Uint64("currTxnCommitTs", currTxnCommitTs),
 			zap.Uint64("lastTxnCommitTs", lastTxnCommitTs),
 			zap.Bool("isLastTime", isLastTime))
-		if w.splitTxn {
-			if currTxnCommitTs == 0 {
-				return
-			}
-			if currTxnCommitTs == lastPos.CommitTs && lastPos.IsCommitFence() {
+		if currTxnCommitTs == lastPos.CommitTs {
+			if lastPos.IsCommitFence() {
 				// All transactions before currTxnCommitTs are resolved.
 				err = w.advanceTableSink(task, currTxnCommitTs, committedTxnSize+pendingTxnSize)
 			} else {
-				// This will advance some complete transactions before currTxnCommitTs,
-				// and one partail transaction with `batchID`.
+				// This means all events of the currenet transaction have been fetched, but we can't
+				// ensure whether there are more transaction with the same CommitTs or not.
 				err = w.advanceTableSinkWithBatchID(task, currTxnCommitTs, committedTxnSize+pendingTxnSize, batchID)
```

Comment:
Does this mean whether splitting txn we all use the batch ID as a marker? Could you please add some comments to batch ID to explain what is going on? How do we use the batch ID for different goals?
Because in the past, batch ID was only for the split txn.

Reply:
Done.